ci/scripts/execute_tests.sh

```diff
@@ -34,6 +34,11 @@ if [[ -z "${GRADLE_TASK}" ]]; then
   exit 1
 fi
 
+. ${SCRIPTDIR}/short_circuit_pr_tasks.sh
+cd geode
+  is_source_from_pr_testable "ci" "dev-tools" "etc" "geode-book" "geode-docs" || exit 0
```

Comment:
any way to avoid duplicating this list two places?

Reply:
If you can think of one, I'm happy to. We could move the directory change into the function itself to make this a one-liner

Reply:
Is it necessary to have the arguments at all given that they are the same in every call? Seems you could whittle this down to a single line `${SCRIPTDIR}/short_circuit_pr_tasks.sh && exit 0`, where `short_circuit_pr_tasks.sh` returns true if it should short circuit.

Reply:
I have little skin in this game but if it doesn't matter than I would make it simple but if its not going to change over time then its probably fine as it is.

Reply:
I also have little skin in the game, but here are some options that jump out at me:

* Like Jake said, let `short_circuit_pr_tasks` be the source of truth for what directories are excluded.
* Take an arg list, and specify those args in the task's `yml`.  Jinja or yml anchoring could reduce duplication.
* Get exceedingly convoluted by having this check be an independent Concourse job which outputs a sentinel file, which test jobs use to determine whether or not to short-circuit to green.

I feel like the first option is best for the first iteration.